src/main/scala/tech/cryptonomic/conseil/util/DatabaseUtil.scala

```diff
@@ -5,18 +5,44 @@ import slick.jdbc.{GetResult, PositionedParameters, SQLActionBuilder}
 import tech.cryptonomic.conseil.generic.chain.DataTypes.AggregationType.AggregationType
 import tech.cryptonomic.conseil.generic.chain.DataTypes.OperationType.OperationType
 import tech.cryptonomic.conseil.generic.chain.DataTypes._
+import tech.cryptonomic.conseil.tezos.Tables
 
 /**
   * Utility functions and members for common database operations.
   */
 object DatabaseUtil {
   lazy val db = Database.forConfig("conseildb")
 
+  trait QueryBuilder[+T <: Table[_]] {
+    def makeQuery(
+        table: String,
+        columns: List[String],
+        aggregation: List[Aggregation]
+    ): SQLActionBuilder
+
+    /* use this to know how the blocks invalidation table is named*/
```

Comment:
this and two other below: lost a space before closing comment

Reply:
Might be the formatter? I don't think it's manual intervention from me